support/dynamics/gp/guidelines-that-support-professionals-use.md

```diff
@@ -16,17 +16,15 @@ _Original KB number:_ &nbsp; 850201
 
 Microsoft support professionals use to determine when a support case becomes a consulting engagement for the following items:
```

Comment:
@lucciz01 please edit this sentence... it is not complete.

Reply:
Hi @aartig13, I moved the first sentence into the Introduction section. You can find my changes in the **Files changed** tab of this PR. Loop @theley502 for further suggestions.

Reply:
Thanks! Looks good